autogen.sh

```diff
@@ -1,5 +1,23 @@
 #!/bin/sh
 
+set -e
+
+# Commands this script needs
+needed='rm mkdir autoreconf echo'
+if ! type $needed >/dev/null
```

Comment:
That's what I expected it to do too, but not what I observed on my system. All the output went to `stderr`. Still, it's probably safer to redirect both output streams. I'll update the PR.